dspace-server-webapp/src/main/resources/spring/spring-dspace-addon-validation-services.xml

```diff
@@ -26,4 +26,18 @@
         <property name="itemService" ref="org.dspace.content.ItemServiceImpl"/>
         <property name="uploadConfigurationService" ref="uploadConfigurationService"/>
     </bean>
+
+    <bean name="clarinLicenseValidation" class="org.dspace.app.rest.submit.step.validation.ClarinLicenseResourceValidation" scope="prototype">
+        <property name="name" value="clarin-license"/>
+        <property name="itemService" ref="org.dspace.content.ItemServiceImpl"/>
+    </bean>
+
+    <bean name="licenseValidation" class="org.dspace.app.rest.submit.step.validation.ClarinLicenseDistributionValidation">
+        <property name="name" value="license"/>
+    </bean>
+
+<!--    <bean name="licenseValidation" class="org.dspace.app.rest.submit.step.validation.LicenseValidation">-->
```

Comment:
please remove; or add reason why you need to leave it there commented out

Reply:
I removed it, thank you